### PR TITLE
add support to Django 1.7 apps

### PR DESCRIPTION
**Pull request by jairhenrique**
Add support to https://docs.djangoproject.com/en/1.7/ref/applications/

### CODE REVIEW

**Comment by jairhenrique** [comment]
[Build Status](https://travis-ci.org/jairhenrique/django-health-check)

File: health_check/__init__.py [hunk 1]
```diff
@@ -17,29 +17,45 @@ def autodiscover():
...
-            # could raise NotRegistered and AlreadyRegistered exceptions
-            # (see #8245).
-            plugin_dir._registry = before_import_registry
-
-            # Decide whether to bubble up this error. If the app just
-            # doesn't have an admin module, we can ignore the error
-            # attempting to import it, otherwise we want it to bubble up.
-            if module_has_submodule(mod, 'plugin_health_check'):
-                raise
+    if django.get_version() >= '1.7':
```
Comment (stefanfoulis) [inline health_check/__init__.py:26]: needs proper version comparison

File: conftest.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        INSTALLED_APPS=(
+            'django.contrib.auth',
+            'django.contrib.contenttypes',
+
+            'health_check',
+            'health_check_cache',
+        ),
+    )
+
+    django.setup()
```
Comment (stefanfoulis) [inline conftest.py:30]: this failed for me in any Django<1.7

**Comment by stefanfoulis** [comment]
thank you very much @jairhenrique for the contribution :-)
I'm sorry I prematurely merged it and had to revert it again. Could you re-open or re-submit it please?

**Pull request merged**